src/iexamine.cpp

```diff
@@ -1455,7 +1456,8 @@ void iexamine::bulletin_board( player &p, const tripoint &examp )
 {
     g->validate_camps();
     point omt = ms_to_omt_copy( g->m.getabs( examp.xy() ) );
-    cata::optional<basecamp *> bcp = overmap_buffer.find_camp( omt );
+    // TODO: fix point types
```

Comment:
This is mentioned in the OP.  Basically I had to stop this refactoring somewhere; and the "boundary layer" of this refactoring pass is marked by these comments, where we transform between the "new" and "old" coordinate types, so that I (or others) know where to go back to to continue this effort.